README.md

```diff
@@ -1,8 +1,56 @@
-# Sourcery Stencils
-Sourcery stencil files for generating Vapor 2 boilerplate
+# Sourcery Templates ✨
+[Sourcery](https://github.com/krzysztofzablocki/Sourcery) stencil files for generating Vapor 2 boilerplate.
 
 # Models
-## Annotations
+This collection of templates is related to models and automating their conversions to common Vapor types.
+
+## Model
+|  Key      | Description                                                     |
+| --------- | ----------------------------------------------------------------|
+|  enumName | Generate a Swift enum for MySQL with accessors for allCases.    |
```

Comment:
Maybe `allCases` should be replaced with ".. for a list of all cases". `allCases` makes me think that the property is actually named this way.

Reply:
Consider this comment everywhere where `enumName` is listed.